### ISSUE TITLE
Entry is missing its type ID

### ISSUE THREAD

**Issue by johnj24** [engram-design/Workflow#101]
### Description
When an editor is on a new entry and then presses the "Save draft and submit for review" button, we're seeing the following error: `Entry is missing its type ID`

### Steps to reproduce

1. Start a new entry in a channel that has just the one default entry type
2. Click "Save draft and submit for review"

### Additional info

- Plugin version: 1.4.1
- Craft version: Pro 3.4.2

**Comment by engram-design** [comment]
Can't seem to reproduce this on my end. What happens when the editor account just hits "Save" on the top-right?

**Comment by johnj24** [comment]
Hitting save in the top-right works as expected. It returns to the entry index page with a new entry in the index.

Would you like the stack trace or a video example?

**Comment by engram-design** [comment]
Hmmm. Both if you can provide them?

**Comment by willard** [comment]
same here.. newly updated craft 3.4.3 with latest version of workflow.. 

Entry is missing its type ID.

**Comment by johnj24** [comment]
```
yii\base\InvalidConfigException: Entry is missing its type ID in path/to/vendor/craftcms/cms/src/elements/Entry.php:856
Stack trace:
#0 path/to/vendor/craftcms/cms/src/elements/Entry.php(804): craft\elements\Entry->getType()
#1 path/to/vendor/craftcms/cms/src/base/Element.php(2726): craft\elements\Entry->getFieldLayout()
#2 path/to/vendor/craftcms/cms/src/base/Element.php(1098): craft\base\Element->fieldByHandle('enabledForSite')
#3 path/to/vendor/verbb/workflow/src/controllers/SubmissionsController.php(220): craft\base\Element->__get('enabledForSite')
#4 path/to/vendor/verbb/workflow/src/controllers/SubmissionsController.php(153): verbb\workflow\controllers\SubmissionsController->_setDraftAttributesFromPost(Object(craft\elements\Entry))
#5 path/to/vendor/verbb/workflow/src/controllers/SubmissionsController.php(71): verbb\workflow\controllers\SubmissionsController->_publishDraft()
#6 [internal function]: verbb\workflow\controllers\SubmissionsController->actionUnsavedDraftSubmission()
#7 path/to/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#8 path/to/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#9 path/to/vendor/craftcms/cms/src/web/Controller.php(178): yii\base\Controller->runAction('unsaved-draft-s...', Array)
#10 path/to/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('unsaved-draft-s...', Array)
#11 path/to/vendor/craftcms/cms/src/web/Application.php(290): yii\base\Module->runAction('workflow/submis...', Array)
#12 path/to/vendor/craftcms/cms/src/web/Application.php(558): craft\web\Application->runAction('workflow/submis...', Array)
#13 path/to/vendor/craftcms/cms/src/web/Application.php(269): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#14 path/to/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#15 path/to/public/index.php(21): yii\base\Application->run()
#16 {main}
```

I'll e-mail a link to the screen recording.

**Comment by masonry47** [comment]
Hi there!  We have the exact same issue

Craft 3.4.5
Workflow 1.4.1

We have no other plugins installed and entries save normally if not going through Workflow. 

Thank you for any info!

**Comment by engram-design** [comment]
Should be fixed in [1.4.2](https://github.com/verbb/workflow/releases/tag/1.4.2)
